Sources/Orbit/Components/Collapse.swift

```diff
@@ -0,0 +1,129 @@
+import SwiftUI
+
+struct CollapseButtonStyle: SwiftUI.ButtonStyle {
+
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .contentShape(Rectangle())
+    }
+}
+
+/// Hides long or complex information.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/collapse/)
+public struct Collapse<Header: View, Content: View>: View {
+
+    let header: Header
+    let content: Content
+
+    @State var isExpanded = false
+
+    public var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            SwiftUI.Button {
+                withAnimation(.easeInOut(duration: 0.2)) {
+                    isExpanded.toggle()
+                }
+            } label: {
+                HStack(spacing: 0) {
+                    header
+                        .accessibility(.collapseHeader)
+                    Spacer(minLength: .xSmall)
+                    Icon(.chevronDown)
+                        .rotationEffect(isExpanded ? .degrees(180) : .zero)
+                }
+                .padding(.vertical, .small)
```

Comment:
This padding might be too specific. Maybe it could be used only when constructing the `String`-based simple version. That way, a call site that provides custom header content would be fully in charge of any required padding.

A `Card` component embeds some "typical" padding configurations as a part of its layout configuration. For this component, that solution might not be necessary.

Reply:
Ok, set only for `Text` headers.